<launch>


<node pkg="chessboard_publisher" type="publisher" name="chessboard_7_4" output="screen">
  <param name="name" value="chessboard" />
  <param name="grid_x_size" value="8" />
  <param name="grid_y_size" value="5" />
  <param name="rect_x_size" value="0.04" />
  <param name="rect_y_size" value="0.04" />
</node>


</launch>
